### ISSUE TITLE
zoom mouse: support for hotkeys

### ISSUE THREAD

**Issue by knausj85** [talonvoice/talon#422]
With hotkeys supported on all platforms in beta, it would be ideal if there was official support for hotkeys in zoom mouse in some fashion. This would allow use of e.g. a foot pedal in addition to noises.

- [ ] The usual mouse actions should ideally be supported. Probably just exposed as actions in zoom mouse?

1. Left/right click
2. Double click
3. Triple click
4. Moving the cursor

- [ ] It probably makes sense to have a scope or similar that includes (1) which eye tracking mode is enabled, and (2) whether the zoom view is up? This would allow various hotkeys to be made context-specific as appropriate.

**Comment by lunixbochs** [comment]
Yep, as part of the eye tracking updates I want to add actions to everything, and move the pop noise into .talon so that's just `noise(pop): eye.zoom_trigger()` or whatever out of the box (internally).

For mouse clicking, maybe the existing mouse_click action implementations in Talon would dismiss zoom mouse by default. I don't want to make new actions for that.

**Comment by lunixbochs** [comment]
in 0.3.1 we have actions for zoom mouse, which is most of this issue
